Drop-In/src/main/java/com/braintreepayments/api/SelectPaymentMethodParentFragment.java

```diff
@@ -68,6 +73,31 @@ public void transformPage(@NonNull View page, float position) {
         return view;
     }
 
+    @Override
+    public void onResume() {
+        super.onResume();
+        slideUpBottomSheet();
+    }
+
+    private void slideUpBottomSheet() {
```

Comment:
Should we add the same slide up animation on the vaulted cards? Those appear much more quickly than the bottom sheet slide up and might look better with the same animation.

Reply:
Yeah I agree. I created a `ViewPager2Animator` class to make the animation more like the others. `ViewPager2` doesn't have an API to control smooth scroll speed so it has to be custom.